### PR TITLE
#147 fixed cwd issues by having commonly mounted reports folder

### PR DESCRIPTION
**Pull request by Anthony-Mckale**
- added a "eze test -t semgrep" run single tool option
- fixed npm audit --omit issues
- fixed __EZE_CONFIG_FOLDER__ issues

and fixed cwd issues by having a common eze config folder that's always available for docker

### CODE REVIEW

File: eze/cli/commands/projects_commands.py [hunk 1]
```diff
@@ -7,6 +7,7 @@
 from eze.utils.language.utils import get_projects
 
 
+# TODO: move to house keeping
```
Comment (mgonzales-rs) [inline eze/cli/commands/projects_commands.py:10]: you mean as an option inside house-keeping command?

File: eze/plugins/tools/node_npmaudit.py [hunk 2]
```diff
@@ -35,20 +37,14 @@ class NpmAuditTool(ToolMeta):
...
     }
     # https://github.com/npm/cli/blob/latest/LICENSE
     LICENSE: str = """NPM"""
-    # TODO: improve with FROM_NPM ?
+    # TODO: improve with FROM_NPM ? ps docker run node:19.3.0-slim npm --version
     VERSION_CHECK: dict = {
         "FROM_EXE": "npm --version",
         "FROM_DOCKER": {"DOCKER_COMMAND": {"IMAGE_NAME": NPM_DOCKER_IMAGE, "BASE_COMMAND": "npm --version"}},
-        "CONDITION": ">=6",
+        "CONDITION": ">=9",
```
Comment (mgonzales-rs) [inline eze/plugins/tools/node_npmaudit.py:47]: we're doing that for node > 9, and what about node 7?

File: eze/cli/commands/projects_commands.py (hunk 1 above)
Comment (Anthony-Mckale) [reply eze/cli/commands/projects_commands.py:10]: think so, doesn't seem like it needs it's own command, was originally planning a range of project tools and reporters, but decided to go down a simpler auto config route instead

File: tests/plugins/tools/test_semgrep.py [hunk 3]
```diff
@@ -118,8 +117,8 @@ async def test_run_scan__correct_docker_command__std(self, mock_async_subprocess
         }
         relative_report = input_config["REPORT_FILE"]
         absolute_folder = get_current_working_folder()
-
-        expected_cmd = f"docker run --rm -v {absolute_folder}:/src returntocorp/semgrep semgrep scan --json --metrics=off -q . --use-git-ignore -c p/ci -c p/dockerfile -c p/nodejs -c p/javascript -o '{relative_report}' --exclude 'test_*.py' --exclude '*.test.js' --exclude tests --exclude __tests__ --something foo"
+        eze_folder_folder = str(create_absolute_path(".eze"))
```
Comment (mgonzales-rs) [inline tests/plugins/tools/test_semgrep.py:120]: could we skip for naming the redundancy? 'eze_folder' or eze_reports_folder ?

**Review by mgonzales-rs — APPROVED** [review]
Left 2 comments, after that LGTM

File: tests/plugins/tools/test_semgrep.py (hunk 3 above)
Comment (Anthony-Mckale) [reply tests/plugins/tools/test_semgrep.py:120]: done

File: eze/cli/commands/projects_commands.py (hunk 1 above)
Comment (Anthony-Mckale) [reply eze/cli/commands/projects_commands.py:10]: so projects is more about finding the manifest files more than anything else

Comment (mgonzales-rs) [reply eze/cli/commands/projects_commands.py:10]: ok, like a pre-scan then 👍

File: eze/plugins/tools/node_npmaudit.py (hunk 2 above)
Comment (mgonzales-rs) [reply eze/plugins/tools/node_npmaudit.py:47]: but we will be giving support for node 7 still (in parse_report), no?

**Pull request merged**